frontend/src/ui-components/components/NotificationToast.vue

```diff
@@ -60,7 +60,7 @@ export default {
             // if a countdown is set, emit the "close" event after the countdown timer expires
             setTimeout(() => {
                 this.close()
-            }, this.countdown)
+            }, 90000000)
```

Comment:
???

Reply:
my bad, that's how I forced the alert to remain present.

reverted